client/state/posts/counts/reducer.js

```diff
@@ -3,20 +3,27 @@
  */
 import { combineReducers } from 'redux';
 import merge from 'lodash/merge';
+import pick from 'lodash/pick';
+import includes from 'lodash/includes';
+import omit from 'lodash/omit';
 
 /**
  * Internal dependencies
  */
 import {
+	CURRENT_USER_ID_SET,
 	POST_COUNTS_RECEIVE,
 	POST_COUNTS_REQUEST,
 	POST_COUNTS_REQUEST_SUCCESS,
 	POST_COUNTS_REQUEST_FAILURE,
+	POST_COUNTS_RESET_INTERNAL_STATE,
+	POST_DELETE,
+	POSTS_RECEIVE,
 	SERIALIZE,
 	DESERIALIZE
 } from 'state/action-types';
-import { isValidStateWithSchema } from 'state/utils';
 import { countsSchema } from './schema';
+import { createReducer } from 'state/utils';
```

Comment:
>neato!

Props to @gziolo for this one (#4805). Especially nice for managing the schema (de)serialization details.